docs/compile.md

```diff
@@ -96,14 +98,15 @@ The recommended way is to use [xcaddy](https://github.com/caddyserver/xcaddy) to
 ```console
 CGO_ENABLED=1 \
 XCADDY_GO_BUILD_FLAGS="-ldflags='-w -s' -tags=nobadger,nomysql,nopgx" \
-CGO_CFLAGS=$(php-config --includes) \
-CGO_LDFLAGS="$(php-config --ldflags) $(php-config --libs)" \
+CGO_CFLAGS="$(php-config --includes) -I$PWD/../../dist/dependencies/include" \
+CGO_LDFLAGS="$(php-config --ldflags) $(php-config --libs) -L$PWD/../../dist/dependencies/lib" \
 xcaddy build \
     --output frankenphp \
     --with github.com/dunglas/frankenphp/caddy \
     --with github.com/dunglas/mercure/caddy \
-    --with github.com/dunglas/vulcain/caddy
-    # Add extra Caddy modules and FrankenPHP extensions here
```

Comment:
@dunglas What is meant by this? I'm not aware of xcaddy being able to install frankenphp extensions.

Reply:
I suspect this is what Kévin meant [on his linkedin post](https://www.linkedin.com/posts/dunglas_php-activity-7333035357594132480-0LUt?utm_source=share&utm_medium=member_desktop&rcm=ACoAAANMtdQB7i0iQnh3JzC9Tf8_MHLXN6-tyZo)

Reply:
Sure, I'm waiting with anticipation ;)

Reply:
I've added it in all languages, thanks to AI.